src/lib/divider/divider.spec.ts

```diff
@@ -0,0 +1,99 @@
+import {async, TestBed} from '@angular/core/testing';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {MatDividerModule} from './divider-module';
+
+
+describe('MatDivider', () => {
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [MatDividerModule],
+      declarations: [
+        MatDividerBase,
+        MatDividerVertical,
+        MatDividerInset,
+        MatDividerVerticalInset,
+        MatDividerWithCssClass,
+      ],
+    });
+
+    TestBed.compileComponents();
+  }));
+
+  it('should not apply any additional class to a list without lines', () => {
```

Comment:
This test is not necessary as you are testing the functionality of the angular framework rather than the created component.

Same for the test below `it('should not clear custom classes provided by user', () => {`

Reply:
I got that test from the original list tests, shown [here](https://github.com/angular/material2/blob/20a23f16e8ac1ced29172709cc837fa500b6377b/src/lib/list/list.spec.ts#L89). I can remove both, but should I replace these tests with others, or are the remaining tests sufficient?